packages/vitest-pool-workers/package.json

```diff
@@ -27,13 +27,13 @@
 	"license": "MIT",
 	"exports": {
 		".": {
-			"import": "./dist/pool/index.mjs",
-			"types": "./types/cloudflare-test.d.ts"
+			"types": "./types/cloudflare-test.d.ts",
```

Comment:
This change is in its own commit, and just removes warnings from the build of the vitest-pool-workers